requirements.txt

```diff
@@ -4,7 +4,11 @@ psutil
 
 # Python linter and formatter
 click==8.0.2 # Our version of black needs an older version of click (https://stackoverflow.com/questions/71673404/importerror-cannot-import-name-unicodefun-from-click)
+<<<<<<< dependabot/pip/black-24.3.0
+black==24.3.0
+=======
 black==21.4b0
+>>>>>>> master
```

Comment:
It looks like the CI failed since there appears to be a merge conflict that was pushed to this branch. @vaughnbetz Would you like me to resolve this rebase.

One thing about this change though is that wintermute cannot support black-24.3.0 yet. When you merge this, wintermute may not be able to work anymore. I can confirm if you would like; however, we may want to upgrade wintermute before merging this change in. I do think this is a good change since it seems to be a security issue.

Reply:
Thanks Alex. If you can resolve the conflict that would be great. I'll hold off on merging until wintermute is sorted.

Reply:
I have resolved the merge conflict.